cpp/versions.json

```diff
@@ -0,0 +1,10 @@
+
+{
+  "packages" : {
+    "Thrust" : {
+      "version" : "1.17.0.0-rc2",
```

Comment:
The version should be `1.17.0` ( or maybe `1.17.0.0` if the tweak fix isn't in 1.17). But you can't have `-rc2` in the `VERSION` entry for a cmake `find_package` call which this does.